### PR TITLE
Highlighting github enterprise

### PR DESCRIPTION
**Pull request by rodolfoBee**
Changing the highlight for GitHub enterprise steps as it is" hidden" in the steps for Github.

### CODE REVIEW

**Comment by rodolfoBee** [comment]
Might also need to change the "Configure" highlight to match, what do you think?

**Review by armenzg — APPROVED** [review]
Please add a before and after screenshot in the PR for posterity. Thanks.

File: src/docs/product/integrations/source-code-mgmt/github/index.mdx [hunk 1]
```diff
@@ -37,7 +37,7 @@ Sentry owner, manager, or admin permissions, and GitHub owner permissions are re
 
 The GitHub integration is available for all projects under your Sentry organization. You can connect multiple GitHub organizations to one Sentry organization, but you **cannot** connect a single GitHub organization to multiple Sentry organizations.
 
-### GitHub Enterprise
+# GitHub Enterprise
```
Comment (shanamatthews) [inline src/docs/product/integrations/source-code-mgmt/github/index.mdx:40]:
```suggestion
## GitHub Enterprise
```

If you make this an L1 heading, it won't show up in the docs sidebar:
Screenshot 2023-04-19 at 10 22 35 AM

**Review by shanamatthews — CHANGES_REQUESTED** [review]

**Review by shanamatthews — APPROVED** [review]

**Comment by shanamatthews** [comment]
@rodolfoBee I'm gonna go ahead and give this a courtesy merge.

**Pull request merged**